Cargo.toml

```diff
@@ -22,6 +22,14 @@ tests_outside_test_module = "warn"
 assertions_on_result_states = "warn"
 error_impl_error = "warn"
 
+[profile.test]
+# Some unit tests create files from raw fds and
+# this triggers UB checks to panic. There is no
+# separate flag to disable those as of right now,
+# so we have to disable debug-assertions.
+# Tracking issue: https://github.com/rust-lang/rust/issues/123499
+debug-assertions = false
```

Comment:
In rust-vmm we fixed these double-close cases by just std::mem::forget-ing the already closed File objects (or just doing into_raw_fd() on them). I don't think we should disable debug assertions.